### DISCUSSION TITLE
显示交易存在被骗风险，是怎么回事,蓝灯到底还能用吗？

### DISCUSSION THREAD

**Discussion by GithubLexi666** [getlantern/lantern-forum-cn#875] [Q&A]
![Uploading 001318e8c4f85e60fe6dcb0ee0f4f21.jpg…]()
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@GithubLexi666 您好. 您放心吧. 因为支付宝的问题所以我们需要国内第三方来帮忙付款. 所以系统自动显示交易存在被骗风险. 这只是系统自动标识而已. 没有问题的. 对了, 请您发这个截图给我吧. 发邮件给我. 谢谢! 
mike@getlantern.org

**Comment by MaxMKMax** [comment]
@GithubLexi666 请问您是在哪一个页面看到这个提示呢?

**Comment by GithubLexi666** [comment]
就是支付的时候看见的

---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年09月19日 00:20 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | GithubLexi666 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 显示交易存在被骗风险，是怎么回事,蓝灯到底还能用吗？ (Discussion #875) |

@GithubLexi666 您好. 您放心吧. 因为支付宝的问题所以我们需要国内第三方来帮忙付款. 所以系统自动显示交易存在被骗风险. 这只是系统自动标识而已. 没有问题的. 对了, 请您发这个截图给我吧. 发邮件给我. 谢谢!
***@***.***
